RecoLocalCalo/HcalRecAlgos/interface/HcalSimpleRecAlgo.h

```diff
@@ -76,11 +79,13 @@ class HcalSimpleRecAlgo {
   HORecHit reconstruct(const HODataFrame& digi,  int first, int toadd, const HcalCoder& coder, const HcalCalibrations& calibs) const;
   HcalCalibRecHit reconstruct(const HcalCalibDataFrame& digi,  int first, int toadd, const HcalCoder& coder, const HcalCalibrations& calibs) const;
 
-  void setpuCorrMethod(int method){ puCorrMethod_ = method; if( puCorrMethod_ ==2 ) psFitOOTpuCorr_ = std::auto_ptr<PulseShapeFitOOTPileupCorrection>(new PulseShapeFitOOTPileupCorrection()); }
+  void setpuCorrMethod(int method){ puCorrMethod_ = method; if( puCorrMethod_ ==2 ) psFitOOTpuCorr_ = std::auto_ptr<PulseShapeFitOOTPileupCorrection>(new PulseShapeFitOOTPileupCorrection()); if( puCorrMethod_ ==3 /*|| puCorrMethod_ == 4*/) hltOOTpuCorr_ = std::auto_ptr<HLTv2>(new HLTv2());}
```

Comment:
please use multiple lines instead of continuing on a very long line, for readability